### DISCUSSION TITLE
Helpppp!! I want to use the last version with door sensors

### DISCUSSION THREAD

**Discussion by IxsharpxI** [SecKatie/ha-wyzeapi#201] [Ideas]
Can anyone help me update the last version that supports door sensors with the necessary version identification so that HA will load it again?

**Comment by SecKatie** [comment]
The latest version *reportedly* (I haven't tested it) works with the v1 sensors.

**Reply by IxsharpxI** [reply]
really i thought they were not supported after the version that said they were not anymore.. and another install of a version after that got rid of them. so if that is true it would be news to me

**Reply by IxsharpxI** [reply]
> The latest version _reportedly_ (I haven't tested it) works with the v1 sensors.

I just updated to the latest and get errors for the old sensors

**Reply by SecKatie** [reply]
I am the maintainer... Anyway can you post the errors?

**Reply by IxsharpxI** [reply]
so its not a specific error in the logs. if i can increase logging level i will if if you tell me how. I think the wyzeapi add on loads fine. but none of my v1 sensors work and just show as unavailable

**Reply by IxsharpxI** [reply]
oops! my bad.. apparently it did load the sensors and new entity id were made so the old ones didnt work.

thanks for all the work!
